src/serialbox-python/compare/compare.py

```diff
@@ -0,0 +1,404 @@
+#!/usr/bin/python3
```

Comment:
On Kesch this fails
Works with -> `#!/usr/bin/env python3`

Reply:
done